SUPESC-606 Add note about Availability Notification deploy file configuration (#1721)
spryker/spryker-docs: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md

];
```

> To enable the new endpoint, adjust a Deploy file:

Reviewer: We are not enabling a new endpoint.

> 2.To set YVES host environment variables, adjust a Deploy file:

Reviewer 2: Use the real Deploy file name.
```suggestion
2. To set YVES host environment variables, adjust the `deply.*.yml` file:
```

Author: We have many deploy files. Also customer can have it's own deploy file names

> 2.To set YVES host environment variables, adjust a Deploy file:

Reviewer 2: ```suggestion
2.To set YVES host environment variables, adjust a deploy file:
```
